vtr_flow/arch/multi_die/README.md

```diff
@@ -0,0 +1,39 @@
+# 3D FPGA Architectures
+
+This directory contains architecture files for 3D FPGAs. The architectures are divided into three sub-directories:
+
+1. **koios_3d:**
+- Contains architecture files based on the [k6FracN10LB_mem20K_complexDSP_customSB_22nm](../COFFE_22nm/k6FracN10LB_mem20K_complexDSP_customSB_22nm.clustered.xml) architecture, utilized in Koios benchmarks.
+- Inside the architecture file, the fabric with multiple sizes based on the sector size is defined.
+- Routing resource and switch delays in this architecture are configured for 7 nm technology, with a delay of 73 ps.
```

Comment:
Does this mean the 3D routing resource and switch delays?  I assume that all switch delays aren't 73 ps.  But all are probably 7 nm.  Please clarify ...

Reply:
The delay of all switches are based on 7 nm (extracted from COFFE). The inter-die connection delay is 73 ps.